src/celery.py

```diff
@@ -0,0 +1,18 @@
+from celery import Celery
+from .config import Config
+
+config = Config()
+app = Celery('tasks',
+             broker=config.broker,
+             backend=config.backend,
```

Comment:
Don't actually need the backend if not returning results from the message call, which we won't be for our run report tasks. But this is useful for testing that tasks can run.